analysis/import/SonarImporter/src/main/java/de/maibornwolff/codecharta/importer/sonar/dataaccess/SonarMetricsAPIDatasource.java

```diff
@@ -59,17 +63,36 @@ public SonarMetricsAPIDatasource(String user, URL baseUrl) {
         this.baseUrl = baseUrl;
     }
 
-    public List<MetricObject> getAvailableMetrics() {
+    public List<String> getAvailableMetricsList() {
```

Comment:
does this really return a list of metrics? Shouldn't the name be getAvailableMetricKeys?